tests/docker/bionic/Dockerfile

```diff
@@ -161,6 +161,4 @@ RUN cd /tmp \
 ENV GNUTLS_NEXT_CLI=/usr/local/gnutls-3.7.2/bin/gnutls-cli
 ENV GNUTLS_NEXT_SERV=/usr/local/gnutls-3.7.2/bin/gnutls-serv
 
-RUN pip3 install --no-cache-dir \
-    mbed-host-tests \
-    mock
+RUN scripts/min_requirements.py
```

Comment:
Hmm, sorry if I'm missing the obvious, but how is that script available from inside the container?